workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl

```diff
@@ -38,31 +67,17 @@
 					<button
 						aria-label="Expand Documentation Menu" class="btn expand-btn" onclick="javascript:;"
 						title="Expand Documentation Menu" type="button">
-						<@clay["icon"] symbol="angle-down-small" />
+							<@clay["icon"] symbol="angle-down-small" />
```

Comment:
remove 1 tab

Reply:
This element is inside button, so I believe should be a tab